ci-jobs/scripts/build-webdriveragents.js

```diff
@@ -0,0 +1,47 @@
+const buildWebDriverAgent = require('./build-webdriveragent');
+const { asyncify } = require('asyncbox');
+const { fs, logger } = require('appium-support');
+const { exec } = require('teen_process');
+
+const log = new logger.getLogger('WDA Build');
+
+async function buildAndUploadWebDriverAgents () {
+  // Get all xcode paths from /Applications/
+  const xcodePaths = (await fs.readdir('/Applications/'))
+    .filter((file) => file.toLowerCase().startsWith('xcode_'));
+
+  // Determine which xcodes need to be skipped
+  let excludedXcodeArr = (process.env.EXCLUDE_XCODE || '').split(',');
+  log.info(`Will skip xcode versions: '${excludedXcodeArr}'`);
+
+  for (let xcodePath of xcodePaths) {
+    if (xcodePath.includes('_beta')) {
+      continue;
+    }
+    // Build webdriveragent for this xcode version
+    log.info(`Running xcode-select for '${xcodePath}'`);
+    await exec('sudo', ['xcode-select', '-s', `/Applications/${xcodePath}/Contents/Developer`]);
+    const xcodeVersion = xcodePath.replace('.app', '').split('_', 2)[1];
```

Comment:
path.parse(xcodePath).name

Reply:
That won't get the version... I want to parse `version` from `xcode_<version>.app`

e.g.) `xcode_10.2.app` should parse as `10.2`

Reply:
nvm I understand now... changed it to `path.parse(xcodePath).split('_', 2)[1]`